genai-on-vertex-ai/natural_language_to_sql/natural_language_to_sql.ipynb

```diff
@@ -0,0 +1,1654 @@
+{
```

Comment:
<p>Please clear the output for this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>